t/18_saputils.t

```diff
@@ -78,28 +82,30 @@ Hosts/vmhana02/vhost="vmhana02"');
     ok(($topology_ready == 1), 'healthy cluster leads to the return of 1');
 };
 
-subtest '[check_hana_topology] healthy cluster with custom node_state_match' => sub {
+subtest '[check_hana_topology] healthy cluster with pacemaker older then 2.1.7' => sub {
     my $saputils = Test::MockModule->new('saputils', no_auto => 1);
     $saputils->redefine(record_info => sub { note(join(' ', 'RECORD_INFO -->', @_)); });
 
     my $topology = calculate_hana_topology(input => 'Global/global/cib-time="Thu Feb  1 18:33:56 2024"
 Global/global/maintenance="false"
 Sites/site_b/b="SOK"
 Hosts/vmhana01/remoteHost="vmhana02"
-Hosts/vmhana01/node_state="PAPERINO"
+Hosts/vmhana01/node_state="online"
```

Comment:
This line seems correct to me and about an important part of your change. Could you please confirm I'm getting it right?

`PAPERINO` is no more ok to be used due to this line https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/21264/files#diff-4a40a74cf6048efd08f8d55e2613d752fb683e6dc56fc331da7e8f23ac0af1e9R158

so this is a test about `check_hana_topology` but `PAPERINO` will no more go through `calculate_hana_topology`, or better to say it will go through like `false`.

Am I reading it right?